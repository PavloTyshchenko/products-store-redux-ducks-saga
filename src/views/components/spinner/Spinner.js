import React from 'react'

import './Spinner.css';

const Spinner = () => {
    return (
        <div className="lds-facebook"><div></div><div></div><div></div></div>
    );
};

export default Spinner;
