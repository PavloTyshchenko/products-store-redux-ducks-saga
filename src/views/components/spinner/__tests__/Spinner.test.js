import React from 'react';
import { shallow } from 'enzyme';

import Spinner from '../Spinner';

describe('Spinner Component', () => {

    it('It should render without errors', () => {
        const component = shallow(<Spinner />);
        const wrapper = component.find(`[data-test='spinnerComponent']`);
        expect(wrapper.length).toBe(1);
    });

    test('Should match snapshot', () => {
        const component = shallow(<Spinner />);
        expect(component.debug()).toMatchSnapshot();
    });
});