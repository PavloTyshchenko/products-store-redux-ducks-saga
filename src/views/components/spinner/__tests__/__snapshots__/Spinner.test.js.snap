// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component Should match snapshot 1`] = `
"<div className=\\"lds-facebook\\" data-test=\\"spinnerComponent\\">
  <div />
  <div />
  <div />
</div>"
`;
